acquia/acquia_cms
# ACMS-1705: Update site studio assets management as per new package manager.

**Motivation**
Update site studio assets management as per new package manager.

**Proposed changes**
Updated assets management to new way.

**Alternatives considered**
NA

**Testing steps**
- Install site with ACMS-1705-1 branch
- Export Site Studio keys
- Enable site studio module
- Verify packages imported, images/icons/font is available

**Merge requirements**
